packages/nextjs/src/copy-config-files.test.ts

```diff
@@ -0,0 +1,180 @@
+import mock from 'mock-fs'
+import fs from 'fs'
+import path from 'path'
+import { copyConfigFiles } from './copy-config-files';
+describe('copy-config-files', () => {
+
+  afterEach(() => {
+    mock.restore()
+  })
+
+  it('should copy config files to a project with pages router', () => {
```

Comment:
Thanks for adding tests!

Can we default to using `async/await` instead of `.then()`?

Reply:
👍 Done!